### ISSUE TITLE
nn.functional.sigmoid is deprecated. Use torch.sigmoid instead

### ISSUE THREAD

**Issue by aribornstein** [PyTorchLightning/lightning-flash#254]
## 🐛 Bug

In Mulit Label Image classification I get the following warning.

/opt/conda/lib/python3.7/site-packages/torch/nn/functional.py:1639: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
